app/models/alaveteli_pro/price.rb

```diff
@@ -12,15 +12,15 @@ def self.list
     end
   end
 
-  def self.retrieve(id)
-    key = AlaveteliConfiguration.stripe_prices.key(id)
-    new(Stripe::Price.retrieve(key))
+  def self.retrieve(param)
+    id = AlaveteliConfiguration.stripe_prices.key(param)
+    new(Stripe::Price.retrieve(id))
   rescue Stripe::InvalidRequestError
     nil
   end
 
   def to_param
-    AlaveteliConfiguration.stripe_prices[id] || id
+    AlaveteliConfiguration.stripe_prices.fetch(id)
```

Comment:
Switched to fetch to see if specs fail. Wondering if we have need the `||` or not